README.md

```diff
@@ -1,107 +1,131 @@
 | Validate workflow definitions (Schema)      | :heavy_check_mark: |
-| Validate workflow definitions (Integrity)   | :heavy_check_mark: |
+| Validate workflow definitions (Integrity)   | :no_entry_sign: |
 | Generate workflow diagram (SVG)             | :no_entry_sign:    |
 
+---
 
 ## Releases
-|                              Latest Releases                               | Conformance to spec version |
-|:--------------------------------------------------------------------------:| :---: |
-| [v1.0.0](https://github.com/serverlessworkflow/sdk-go/releases/tag/v1.0.0) | [v0.5](https://github.com/serverlessworkflow/specification/tree/0.5.x) |
-| [v2.0.1](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.0.1) | [v0.6](https://github.com/serverlessworkflow/specification/tree/0.6.x) |
-| [v2.1.2](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.1.2) | [v0.7](https://github.com/serverlessworkflow/specification/tree/0.7.x) |
-| [v2.4.1](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.4.1) | [v0.8](https://github.com/serverlessworkflow/specification/tree/0.8.x) |
 
-## How to use
+|                              Latest Releases                               |                       Conformance to Spec Version                        |
+|:--------------------------------------------------------------------------:|:------------------------------------------------------------------------:|
+| [v1.0.0](https://github.com/serverlessworkflow/sdk-go/releases/tag/v1.0.0) |  [v0.5](https://github.com/serverlessworkflow/specification/tree/0.5.x)  |
+| [v2.0.1](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.0.1) |  [v0.6](https://github.com/serverlessworkflow/specification/tree/0.6.x)  |
+| [v2.1.2](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.1.2) |  [v0.7](https://github.com/serverlessworkflow/specification/tree/0.7.x)  |
+| [v2.4.1](https://github.com/serverlessworkflow/sdk-go/releases/tag/v2.4.1) |  [v0.8](https://github.com/serverlessworkflow/specification/tree/0.8.x)  |
+| [v3.0.0](https://github.com/serverlessworkflow/sdk-go/releases/tag/v3.0.0) | [v1.0.0](https://github.com/serverlessworkflow/specification/releases/tag/v1.0.0-alpha5) |
+
+---
 
-Run the following command in the root of your Go's project:
+## Getting Started
 
-```shell script
-$ go get github.com/serverlessworkflow/sdk-go/v2
+### Installation
+
+To use the SDK in your Go project, run the following command:
+
+```shell
+$ go get github.com/serverlessworkflow/sdk-go/v3
 ```
 
-Your `go.mod` file should be updated to add a dependency from the Serverless Workflow specification.
+This will update your `go.mod` file to include the Serverless Workflow SDK as a dependency.
 
-To use the generated types, import the package in your go file like this:
+Import the SDK in your Go file:
 
 ```go
-import "github.com/serverlessworkflow/sdk-go/v2/model"
+import "github.com/serverlessworkflow/sdk-go/v3/model"
```

Comment:
is there a need to have a migration guide?

Reply:
Good question! But no, people will get to it with the new DSL. In the README, we have the supporting matrix. The parser API remains the same.